.github/CONTRIBUTING.md

```diff
@@ -72,19 +72,20 @@ We follow a "typical" GitHub workflow for contributing changes.
       case**.
     * Commit message contents don't need a lot of details, but make sure others
       can look back later and understand your changes and why.
-4. **[Create](https://help.github.com/articles/creating-a-pull-request/) a new
+4. Ensure that you follow the [pul request requirements](#pull-request-requirements).
```

Comment:
Sooooo close! Minor typo: `pul` => `pull`

Reply:
Oof, and I thought to myself "well it's such a minor change so I don't need to verify it!". Lesson learned.

Reply:
Amended.